ister_gui.py

```diff
@@ -713,7 +714,19 @@ def handler(self, config):
     def build_ui_widgets(self):
         """Build ui handler
         The last urwid Divider helps the tab movement to work"""
+        if os.environ.get('https_proxy'):
+            https_text = os.environ['https_proxy']
+        else:
+            https_text = ''
+
+        if os.environ.get('http_proxy'):
+            http_text = os.environ['http_proxy']
+        else:
+            http_text = ''
+
```

Comment:
Getting proxy information from the environment seems fine but I'm not sure that's really useful for an autorun installer where it shouldn't ever really be set. Is there some use case I'm missing?